qiskit/utils/mitigation/circuits.py

```diff
@@ -114,10 +114,10 @@ def complete_meas_cal(
 
 def tensored_meas_cal(
     mit_pattern: List[List[int]] = None,
-    qr: Union[int, List["QuantumRegister"]] = None,
-    cr: Union[int, List["ClassicalRegister"]] = None,
+    qr: Union[int, List["QuantumRegister"]] = None, # noqa: F821
```

Comment:
The [docs](https://github.com/charliermarsh/ruff#pyflakes-f) say that F821 is an "undefined name error" -- do you think this is that because the type is in quotes?

Reply:
Yes. That's it. I did not track down the reasoning. Of course, the name is in quotes precisely because the symbol is not defined. Maybe some people don't think this should be done, so there is a rule ?

Reply:
@Cryoris  Just because of this issue, I have reverted the application of this rule in 80b078b967c03d9a64f50accae1c166adf506e91 . There may be a way to sort this out, make a global exception or something. But for now, it's just in the way here.